airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -1937,6 +1949,16 @@ static boolean hasNewVersion(final String currentVersion, final String latestVer
     }
   }
 
+  @VisibleForTesting
+  static boolean hasNewPatchVersion(final String currentVersion, final String latestVersion) {
+    try {
+      return new AirbyteVersion(latestVersion).checkOnlyPatchVersionIsUpdated(new AirbyteVersion(currentVersion));
+    } catch (final Exception e) {
```

Comment:
what exception would be thrown? generally when we catch exceptions it is 1 of 2 special cases: 
1. there is a non-runtime exception we need to catch (in that case we want to catch that specific exception not the broader `Exception`)
2. we don't care if the code fails and always want to move on. 

I don't think this code meets either of these. I think you can just remove the try/catch.